app/views/admin/sdg/managers/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2><%= t("admin.sdg_managers.index.title") %></h2>
+
+<%= render "admin/shared/user_search", url: search_admin_sdg_managers_path %>
+
+<div id="sdg_managers">
+  <% if @sdg_managers.any? %>
+    <h3 class="margin"><%= page_entries_info @sdg_managers %></h3>
+
+    <table>
+      <thead>
+        <th scope="col"><%= t("admin.sdg_managers.index.name") %></th>
```

Comment:
How about editing the YAML translations and add:

```
en:
  attributes:
    name: "Name"
    email: "Email"
```

Then we could remove these keys and use `SDG::Manager.human_attribute_name` instead :thinking:.

This is related to issue #2093.

Reply:
Sorry, I meant at the top of the YAML file, where it says:

```
en:
  attributes:
    geozone_id: "Scope of operation"
    results_enabled: "Show results"
    sdg_target_list: "Targets"
    stats_enabled: "Show stats"
    advanced_stats_enabled: "Show advanced stats"
```

This way we don't have the same translations for the same attributes over and over :relieved:.